README.md

```diff
@@ -1,40 +1,40 @@
 # knausj_talon
+
 Talon configs for Mac, Windows, and Linux. Very much in progress. This is also intended to work with both Dragon Naturally Speaking and wav2letter.
 
-Clone repo into ~/.talon/user/knausj_talon
+Clone the repository into `~/.talon/user/knausj_talon`:
 
     git clone git@github.com:knausj85/knausj_talon.git knausj_talon
 
-It should look like
-
-talon\user\knausj_talon
-
-talon\user\knausj_talon\code
+The folders should look like:
 
-talon\user\knausj_talon\lang
+    talon\user\knausj_talon
+    talon\user\knausj_talon\code
+    talon\user\knausj_talon\lang
 
-All user-defined actions in this repository are prefixed "user." by talon
+All user-defined actions in this repository are prefixed `user.` by Talon.
 
 ## Jetbrains commands
 
 For Jetbrains commands to work you must install https://plugins.jetbrains.com/plugin/10504-voice-code-idea
 into each editor.
 
 ...
+
 ## Windows
-Running Talon as an adminstator is highly recommended. 
+
+Running Talon as an administrator is highly recommended.
```

Comment:
thanks to using uiAccess, talent should not require administrative privileges anymore

are there still other issues?

Reply:
Admin rights are no longer required, so long as you have it in program files or another secure directory.